packages/api/src/main/java/com/incubator/edupayroll/entity/DocumentRow.java

```diff
@@ -0,0 +1,31 @@
+package com.incubator.edupayroll.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+@Entity
+@Table(name = "DocumentRow")
+public class DocumentRow extends BaseEntity {
+
+    @ManyToOne
+    @JoinColumn(name = "teacher_id", referencedColumnName = "id")
+    private Teacher teacher;
+
+    // CourseType enum lazıms
```

Comment:
buradaki yorumu silelim ve şuna bakabilirsin:
https://www.baeldung.com/jpa-persisting-enums-in-jpa